### ISSUE TITLE
[OMP] email_templates_data table missing when upgrading from version 3.1.2-4 to 3.2.0

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#5593]
**Describe the bug**
When running the command `php tools/upgrade.php upgrade` for upgrading an OMP to 3.2 it always get stuck with an error message: 
`ERROR: Upgrade failed: DB: Table 'email_templates_data' doesn't exist`

**To Reproduce**
Download the new OMP version.
Set all parameters on config.inc.php
run `php tools/upgrade.php upgrade`

**What application are you using?**
OMP version 3.1.2-4 => 3.2.0

**Additional information**
See the error log and the database screenshot before to run the script.
```
[data: dbscripts/xml/upgrade/3.2.0_preupdate_email_templates.xml]
[data: dbscripts/xml/upgrade/3.2.0_preupdate_versioning.xml]
[data: dbscripts/xml/upgrade/3.2.0_preupdate_last_activity.xml]
[schema: lib/pkp/xml/schema/common.xml]
[schema: lib/pkp/xml/schema/scheduledTasks.xml]
[schema: lib/pkp/xml/schema/temporaryFiles.xml]
[schema: lib/pkp/xml/schema/log.xml]
[schema: lib/pkp/xml/schema/notes.xml]
[schema: lib/pkp/xml/schema/controlledVocab.xml]
[schema: lib/pkp/xml/schema/metadata.xml]
[schema: lib/pkp/xml/schema/announcements.xml]
[schema: lib/pkp/xml/schema/reviews.xml]
[schema: lib/pkp/xml/schema/reviewForms.xml]
[schema: lib/pkp/xml/schema/submissions.xml]
[schema: lib/pkp/xml/schema/submissionFiles.xml]
[schema: lib/pkp/xml/schema/tombstone.xml]
[schema: lib/pkp/xml/schema/genres.xml]
[schema: lib/pkp/xml/schema/libraryFiles.xml]
[schema: lib/pkp/xml/schema/rolesAndUserGroups.xml]
[schema: lib/pkp/xml/schema/views.xml]
[schema: lib/pkp/xml/schema/metrics.xml]
[schema: lib/pkp/xml/schema/navigationMenus.xml]
[schema: lib/pkp/xml/schema/categories.xml]
[schema: dbscripts/xml/omp_schema.xml]
[code: Installer Installer::installDefaultNavigationMenus]
WARNING: The NavigationMenu (ContextId: 1, Title: User Navigation Menu, Area: user) will be skipped because the specified area has already a NavigationMenu attached.
WARNING: The NavigationMenu (ContextId: 1, Title: Primary Navigation Menu, Area: primary) will be skipped because the specified area has already a NavigationMenu attached.
WARNING: The NavigationMenu (ContextId: 0, Title: User Navigation Menu, Area: user) will be skipped because the specified area has already a NavigationMenu attached.
[code: Installer Installer::migrateStaticPagesToNavigationMenuItems]
[code: Installer Installer::migrateUserAndAuthorNames (skipped)]
[data: dbscripts/xml/upgrade/3.2.0_update.xml]
ERROR: Upgrade failed: DB: Table 'email_templates_data' doesn't exist
```
![image](https://user-images.githubusercontent.com/4992902/76128040-42b23400-5fd1-11ea-80ea-e59431ea1d8d.png)

**Comment by asmecher** [comment]
@israelcefrin, is this a copy of OMP 3.2.0-0 from the `.tar.gz` package, or from github?

**Comment by israelcefrin** [comment]
@asmecher it is from github repo.

**Comment by asmecher** [comment]
Is it possible that this is not quite up to date with the 3.2.0 code? I received this message myself during the last bit of testing/release preparation and resoved it with fafc0974b92d9b7d9cbd88978d939c6db35357e3 and 45b75f126eec89334ead5f26c1f3de473333b9b6, which should be included in the 3.2.0-0 release package.

**Comment by israelcefrin** [comment]
@asmecher  It is passing through this step, however I've found another issue which I will file to track it.
